shoplist/src/main/java/com/netguru/codereview/network/ShopListApi.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.network
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.flow.Flow
+
+interface ShopListApi {
```

Comment:
our interface looks good, it will be helpful while calling database with Room Library or fetching data from a web service.